examples/chip-tool/commands/clusters/Commands.h

```diff
@@ -10702,6 +10723,88 @@ class ReadOnOffClusterRevision : public ModelCommand
 | * ClusterRevision                                                   | 0xFFFD |
 \*----------------------------------------------------------------------------*/
 
+/*
+ * Command AddOpCert
+ */
+class OperationalCredentialsAddOpCert : public ModelCommand
+{
+public:
+    OperationalCredentialsAddOpCert() : ModelCommand("add-op-cert")
+    {
+        AddArgument("noc", &mNoc);
+        AddArgument("iCACertificate", &mICACertificate);
+        AddArgument("iPKValue", &mIPKValue);
+        AddArgument("caseAdminNode", 0, UINT64_MAX, &mCaseAdminNode);
+        AddArgument("adminVendorId", 0, UINT16_MAX, &mAdminVendorId);
+        ModelCommand::AddArguments();
+    }
+    ~OperationalCredentialsAddOpCert()
+    {
+        delete onSuccessCallback;
+        delete onFailureCallback;
+    }
+
+    CHIP_ERROR SendCommand(ChipDevice * device, uint8_t endpointId) override
+    {
+        ChipLogProgress(chipTool, "Sending cluster (0x003E) command (0x06) on endpoint %" PRIu16, endpointId);
+
+        chip::Controller::OperationalCredentialsCluster cluster;
+        cluster.Associate(device, endpointId);
+        return cluster.AddOpCert(
```

Comment:
Add TODO about needing to send the NOC and ICA certs in a CHIP certificate array.